### DISCUSSION TITLE
No he recibido correo después de haber enviado la liga de mis katas.

### DISCUSSION THREAD

**Discussion by roge64** [LaunchX-InnovaccionVirtual/CursoIntroPython#295] [Q&A]
¿Se tardan en enviar el correo? Mandé la liga hace dos días.

**Comment by YomairaBarrera** [comment]
ya revisaste tu correo de innovaccion en Outlook? porque en realidad a mi después de mandar el forms ni un minuto había pasado y ya tenia los correos, además de que automáticamente cuando fue correcto el envió me agrego al team de front end.

**Reply by roge64** [reply]
Si, ni en el que nos hicieron en Outlook ni en el email con el que me registré.
Pero si estoy agregado en el equipo de front end, inclusive desde antes de enviar mis katas.

**Reply by YomairaBarrera** [reply]
Entonces no hay problema, ya estas adentro si hubo casos en los que no les llego el correo pero lo importante es que estas en el team de Front End, lo dijeron en el live

**Reply by MARYCHS** [reply]
Hola como reviso mi correo de launch, por fín logré enviar mis katas y quiero ver si recibí el correo, antes de que se cierre la entrega para el siguiente curso por faaaa

**Reply by MARYCHS** [reply]
si recibí el correo, gracias ya logré entrar!!!

**Comment by AlejandroMontoyaT** [comment]
Todos los correo llegar al de innovacion, revisa tu spam y en el grupo de team revisa en grupos, te debe de aparecer 1 grupo más.
